apps/src/templates/studioHomepages/SectionsAsStudentTable.jsx

```diff
@@ -128,13 +127,6 @@ class SectionsAsStudentTable extends React.Component {
     });
   }
 
-  sectionHref(section) {
```

Comment:
from as far as i can tell, this method is never used, so i'm removing it instead of fixing the URL to the old teacher dashboard